scalableminds/webknossos
# Add beforeunload to dataset upload view

This PR double checks with the user if he/she really wants to leave the dataset upload view. 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- upload a dataset
- try to navigate elsewhere while it's still uploading
- you should be warned about leaving while it's still loading

### Issues:
- fixes #5051

------
